src/rabbitmq.js

```diff
@@ -293,16 +290,13 @@ class RabbitMQ {
    */
   publishEvent (exchange: string, content: Object): Bluebird$Promise<void> {
     return Promise.try(() => {
-      const bufferContent = this._validatePublish(exchange, content, 'events')
-      const jobMeta = {
-        appId: this.name,
-        timestamp: Date.now()
-      }
-      this.log.info({ event: exchange, job: content, jobMeta: jobMeta }, 'Publishing event')
+      this._validatePublish(exchange, content, 'events')
```

Comment:
and then you can use this here and here:
https://github.com/Runnable/ponos/pull/99/files#diff-2457f035d00feb69f8f52297277627edR273